include/aikido/perception/PoseEstimatorModule.hpp

```diff
@@ -75,6 +87,12 @@ class PoseEstimatorModule : public PerceptionModule
 
   /// Listens to the transform attached to the node
   tf::TransformListener mTfListener;
+
+  /// For additional information for each object
```

Comment:
Examples here would be helpful!

Reply:
I agree. I will add some examples of new methods.